apps/pay/app/[username]/page.tsx

```diff
@@ -81,7 +91,7 @@ function ReceivePayment() {
 
   return (
     <>
```

Comment:
```suggestion
```

Reply:
those might typically not be needed (I know they were part of the prior code, but it's a good time to trim those placeholder that are creating code indentation that are just making it more difficult to read the code)

Reply:
> those might typically not be needed (I know they were part of the prior code, but it's a good time to trim those placeholder that are creating code indentation that are just making it more difficult to read the code)

This is needed only single root element can be returned.

Reply:
AFAI see from github, there is still only a single element down the tree: `<Container className={styles.payment_container}>`? in that case, this element should be enough

Reply:
Yup, you are right, I missed it.